src/Land/Model/soil_water.jl

```diff
@@ -8,25 +8,27 @@ abstract type AbstractWaterModel <: AbstractSoilComponentModel end
 Model structure for a prescribed water content model.
 
 The user supplies functions of space and time for both `ϑ_l` and
-`θ_ice`. No auxiliary or state variables are added, no PDE is solved. 
+`θ_i`. No auxiliary or state variables are added, no PDE is solved. 
 The defaults are no moisture anywhere, for all time. 
 
 # Fields
 $(DocStringExtensions.FIELDS)
 """
-struct PrescribedWaterModel{F1, F2} <: AbstractWaterModel
+struct PrescribedWaterModel{FN1, FN2} <: AbstractWaterModel
     "Augmented liquid fraction"
-    ϑ_l::F1
+    ϑ_l::FN1
     "Volumetric fraction of ice"
-    θ_ice::F2
+    θ_i::FN2
 end
 
 """
-    PrescribedWaterModel(
-        ϑ_l::Function = (aux,t) -> eltype(aux)(0.0),
-        θ_ice::Function = (aux,t) -> eltype(aux)(0.0),
-    ) where {FT}
-
+    function PrescribedWaterModel(
+        ϑ_l::Function = (aux, t) -> eltype(aux)(0.0),
+        θ_i::Function = (aux, t) -> eltype(aux)(0.0),
+    )
+        args = (ϑ_l, θ_i)
+        return PrescribedWaterModel{typeof.(args)...}(args...)
+    end
 Outer constructor for the PrescribedWaterModel defining default values, and
 making it so changes to those defaults are supplied via keyword args.
```

Comment:
hey! another kind of picky thing. The functions are no longer given with kwargs, so we can just say
"Outer constructor for the PrescribedWaterModel defining default values."

Reply:
got it :)